service/src/main/java/org/ehrbase/dao/access/jooq/FolderAccess.java

```diff
@@ -142,9 +142,8 @@ private Boolean update(final Timestamp transactionTime,
         // Copy into new instance and attach to DB context.
         // The new instance is required to store the new record with a new ID
         FolderRecord updatedFolderRecord = new FolderRecord();
-        if (rootFolder) {
-            updatedFolderRecord.setId(this.getFolderId());
-        }
+        updatedFolderRecord.setId(this.getFolderId());
```

Comment:
why do you want to maintain the id of old subfolders that are actually complete  new folders that override the old ones?